components/com_users/tmpl/registration/default.php

```diff
@@ -34,10 +35,19 @@
 					<?php if (isset($fieldset->label)) : ?>
 						<legend><?php echo Text::_($fieldset->label); ?></legend>
 					<?php endif; ?>
-					<?php echo $this->form->renderFieldset($fieldset->name); ?>
+					<?php foreach ($fields as $field): ?>
+						<?php if ($field->name=="jform[captcha]"): ?>
+							<?php $captcha=$field; ?>
+						<?php else: ?>
+							<?php echo $field->renderField();?>
+						<?php endif; ?>		
+					<?php endforeach; ?>
 				</fieldset>
 			<?php endif; ?>
 		<?php endforeach; ?>
+		<?php if($captcha): ?>
```

Comment:
```suggestion
		<?php if ($captcha): ?>
```
Code style: One space between if and the opening bracket for the condition.